src/stores/applets.js

```diff
@@ -56,6 +55,8 @@ import api from '@/api'
  */
 
 export const useAppletsStore = defineStore('applets', () => {
+  /** @type {object} */
+  const appletApi = ref(undefined)
   const useAlertStore = ref(undefined)
```

Comment:
Yeah, I was kind of trying to sidestep this situation because it got unfavorable fast. 
I can put the alertStore back in, but the issue is that it fractures the CbAlert component and does absolutely nothing on the HUI (cause the rest of the CbAlert is needed in order to display anything) (thus the additional console.log substitution)

I did initially try and move all of CbAlerts over but then it gets complicated with vue-router being so integral. And while i was trying to unravel that it felt like I was several steps away from the thing I was actually trying to do in the HUI (which again, doesn't //can't? use CbAlerts )

Reply:
Gotcha. In that case, let's just strip out alerts from applets. I think the only two things that alerts do are:
 - report that applets are loading
   - I don't think we actually use the loading indicator, so let's just ignore that
 - report that applets failed to load
   - option 1: could add a "fetchFailure" attr to the store and shift alert responsibility to the CUI to respond to that changing prop.
   - option 2: could throw an error in fetchApplets that CbAppletTarget catches, which fires a vue "fetchFailed" event (I like something like this)
   - option 3: Whatever else you want to do

Reply:
using event emitters (option2)